include/seqan/index/index_fm.h

```diff
@@ -205,6 +205,16 @@ class Index<TText, FMIndex<TSpec, TConfig> >
     typename Fibre<Index, FibreLF>::Type            lf;
     typename Fibre<Index, FibreSA>::Type            sa;
 
+    /*!
+     * @fn FMIndex::Index
+     * @brief Constructor
+     *
+     * @signature Index::Index();
+     * @signature Index::Index(text);
+     *
+     * @param[in] text The text to be indexed.
+     */
```

Comment:
@h2 @cpockrandt  is it true, that the text must be in reversed for the FM-Index. If so we should drop a node in the description of the descriptor.